java/core/page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/PageFactory.java

```diff
@@ -29,7 +26,8 @@
     private static Actions actions;
     private static PageManager pageManager;
     private static VideoRecorder videoRecorder;
-    private static boolean aspectsDisabled = false;
+    private static boolean aspectsEnabled = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "aspectsEnabled" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "aspectsEnabled" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)